File: java/evento.py
from compromisso import Compromisso

class Evento(Compromisso):
    def __init__(self, id, data, hora, duracao, prioridade, nomeEvento, local):
        super().__init__(id, 'E', data, hora, duracao, prioridade, nomeEvento)
        
        self.local = local
        self.calcularPrioridadeFinal()
    
    def calcularPrioridadeFinal(self):
        self.prioridadeFinal = self.prioridade * 3
    
    def getDescricao(self):
        return self.descricao
    
    def isAdiavel(self):
        return False
    
    def getLocal(self):
        return self.local

File: java/Evento.java
public class Evento extends Compromisso{
    private String local;

    public Evento(int id, Data data, Hora hora, int duracao, int prioridade, String nomeEvento, String local){
        super(id, 'E', data, hora, duracao, prioridade, nomeEvento);
        this.local = local;
        calcularPrioridadeFinal();
    }

    @Override
    public void calcularPrioridadeFinal(){
        //multiplicador 3x
        this.prioridadeFinal = this.prioridade * 3;
    }
    

    @Override
    public String getDescricao(){
        return this.descricao;
    }

    @Override
    public boolean isAdiavel(){
        //eventos n sao adiaveis
        return false;
    }

    public String getLocal(){
        return local;
    }
}